assets/queries/terraform/gcp/vm_serial_ports_are_enabled_for_vm_instances/metadata.json

```diff
@@ -2,8 +2,8 @@
   "id": "97fa667a-d05b-4f16-9071-58b939f34751",
   "queryName": "Serial Ports Are Enabled For VM Instances",
   "severity": "MEDIUM",
-  "category": "Insecure Configurations",
-  "descriptionText": "Check if VM instance enables serial ports",
+  "category": "Networking and Firewall",
+  "descriptionText": "VM instance should not enable serial ports",
```

Comment:
Please, also update in `kics\assets\queries\ansible\gcp\serial_ports_enabled_for_vm_instances\metadata.json`